pkgs/development/ocaml-modules/printbox/default.nix

```diff
@@ -1,19 +1,19 @@
-{ lib, fetchFromGitHub, buildDunePackage, ocaml, mdx }:
+{ lib, fetchFromGitHub, buildDunePackage, ocaml, mdx, uucp, uutf }:
 
 buildDunePackage rec {
   pname = "printbox";
-  version = "0.2";
+  version = "0.3";
 
   minimumOCamlVersion = "4.05";
 
   src = fetchFromGitHub {
     owner = "c-cube";
     repo = pname;
-    rev = version;
-    sha256 = "16nwwpp13hzlcm9xqfxc558afm3i5s802dkj69l9s2vp04lgms5n";
+    rev = "v${version}";
```

Comment:
Is this change correct? Here is a URL from which the source can be fetched: https://github.com/c-cube/printbox/releases/download/0.3/printbox-0.3.tbz

Reply:
It seems that the revision at upstream has changed after the PR was first written. Fixed now.